portfolio.css

```diff
@@ -406,4 +406,4 @@ form > ul {
   padding: 12px 16px;
   background: #fff;
   border-radius: 8px;
-}
\ No newline at end of file
+}
```

Comment:
- [ ] as requested by the previous reviewer  kindly try to implement button interactions such as hover, active .; etc as shown in Figma design

![192822959-9f0ccbd4-4720-4afd-a81c-7f293746ffbf](https://user-images.githubusercontent.com/89282221/192847281-382fcf67-f62b-404f-aef7-90f5039e3aa9.png)